# Standard Header
std_msgs/Header header

# Array of `object_stamped` topics
yolov7_ros/Object[] objects
